src/constants.ts

```diff
@@ -302,9 +302,9 @@ export const EMAIL_ALREADY_EXISTS_ERROR = {
 
 export const BASE_URL = `http://localhost:${process.env.port || 4000}/`;
 
-export const ACCESS_TOKEN_SECRET = process.env.ACCESS_TOKEN_SECRET;
+export const ACCESS_TOKEN_SECRET = `${process.env.ACCESS_TOKEN_SECRET || ""}`;
```

Comment:
These environment variables should be parsed/validated/transformed/type-asserted booting up the server. Something like zod or json schema would be a good tool to do this.

Reply:
Using `zod` is a very brilliant idea to tackle this issue. We should use it to validate the process.env variables as well. I will give it a try